timescaledb/quick-start/golang.md

```diff
@@ -1,293 +1,266 @@
+other tasks can and should all be executed on the hypertable.
 
-Virtually all user interactions with TimescaleDB are with hypertables.
-Creating tables and indexes, altering tables, inserting data, selecting
-data, etc. can (and should) all be executed on the hypertable.
+<procedure>
 
-A hypertable is defined by a standard schema with column names and
-types, with at least one column specifying a time value.
+<Collapsible heading="Generating a hypertable" headingLevel={3}>
 
-### Step 1: Formulate CREATE TABLE SQL Statements for hypertable
+1.  Create a variable for the `CREATE TABLE SQL` statement for your hypertable.
+    Notice how the hypertable has the compulsory time column:
 
-First, we create a variable which houses our `CREATE TABLE SQL` statement
-for our hypertable. Notice how the hypertable has the compulsory time column:
+    ```go
+    FIXME
+    ```
 
-Second, we formulate the SELECT statement to convert the table created
-into a hypertable. Note that we must specify the table name which we
-wish to convert to a hypertable and its time column name as the two
-arguments, as mandated by the [create_hypertable docs][hypertable-docs]:
+1.  Formulate the `SELECT` statement to convert the table into a hypertable. You
+    must specify the table name to convert to a hypertable, and its time column
+    name as the two. For more information, see the
+    [`create_hypertable` docs][create-hypertable-docs]:
 
-```go
-queryCreateHypertable := `CREATE TABLE sensor_data (
-       time TIMESTAMPTZ NOT NULL,
-       sensor_id INTEGER,
-       temperature DOUBLE PRECISION,
-       cpu DOUBLE PRECISION,
-       FOREIGN KEY (sensor_id) REFERENCES sensors (id)
-       );
-       SELECT create_hypertable('sensor_data', 'time');
-       `
-```
-
-### Step 2: Execute SQL statement using .Exec()
-
-Next, we execute our `CREATE TABLE` statement and `SELECT` statement which
-converts the table created into a hypertable. We do this by calling
-the `Exec()` function on the dbpool object, using the arguments of the
-current context and our statement string `queryCreateHypertable` formulated
-in step 1 above.
-
-```go
-package main
-
-import (
-   "context"
-   "fmt"
-   "os"
-
-   "github.com/jackc/pgx/v4"
-   "github.com/jackc/pgx/v4/pgxpool"
-)
-
-func main() {
-
-   ctx := context.Background()
-   connStr := "yourConnectionStringHere"
-   dbpool, err := pgxpool.Connect(ctx, connStr)
-   if err != nil {
-       fmt.Fprintf(os.Stderr, "Unable to connect to database: %v\n", err)
-       os.Exit(1)
-   }
-   defer dbpool.Close()
-
-   /********************************************/
-   /* Create Hypertable                        */
-   /********************************************/
-   // Create hypertable of time-series data called sensor_data
-
-   //formulate statement
-   queryCreateHypertable := `CREATE TABLE sensor_data (
+    ```go
+    queryCreateHypertable := `CREATE TABLE sensor_data (
```

Comment:
I think this code block is mixing up the previous step and the current step. The last line belongs to this step, and the rest belongs to the previous step.

Reply:
@jonatas ?

Reply:
Yes! that was a mess. I think I resolved the issue now.

Reply:
Just to make it clear, it starts exposing how the command will be and later it wraps up the same code block into a larger context that is runnable.

I think this is right now.